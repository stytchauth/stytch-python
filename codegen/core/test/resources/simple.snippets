snippet1=hello world
