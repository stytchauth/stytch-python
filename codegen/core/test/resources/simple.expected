value = 123
# MANUAL(snippet1)
hello world
# ENDMANUAL(snippet1)
value2 = 456
